titan/features/haart.py

```diff
@@ -80,9 +74,9 @@ def update_agent(self, model: "model.TITAN"):
             model: the instance of TITAN currently being run
         """
         if (
-            self.agent.hiv.active  # type: ignore[attr-defined]
-            and self.agent.hiv.dx  # type: ignore[attr-defined]
-            and model.time >= model.params.hiv.start_time
+            self.agent.hiv.dx  # type: ignore[attr-defined]
+            and model.time
+            >= model.params.hiv.start_time  # TO_REVIEW why based on hiv start time - should there be a haart start time?
```

Comment:
based on HIV start time because we've never had them not paired. It doesn't make sense to have haart before HIV, and I'm not sure the utility of starting it after (unless I suppose timeline scaling-aligned stuff)